package.json

```diff
@@ -33,7 +33,9 @@
     "hammerjs": "2.0.8",
     "helmet": "3.6.1",
     "highcharts": "5.0.12",
+    "history": "4.6.3",
     "keythereum": "1.0.0",
+    "lodash": "^4.17.4",
```

Comment:
Two comments:
1. We have a long-standing rule that "kitchen sink" libraries like lodash are not permitted.  Can you import whatever methods you need in isolation, rather than the entire library?
2. Should use a fixed version number for all dependencies.